source/common/access_log/access_log_formatter.cc

```diff
@@ -498,6 +498,11 @@ StreamInfoFormatter::StreamInfoFormatter(const std::string& field_name) {
         return UnspecifiedValueString;
       }
     };
+  } else if (field_name == "NOOP_ROUTE_NAME") {
```

Comment:
Needs documentation. Also needs implementation in gRPC access logs if we decide to go this route.